users.go

```diff
@@ -390,39 +397,37 @@ func (s *UsersService) DeleteSSHKeyForUser(user int, kid int) (*Response, error)
 // BlockUser blocks the specified user. Available only for admin.
 //
 // GitLab API docs: http://doc.gitlab.com/ce/api/users.html#block-user
-func (s *UsersService) BlockUser(user int) (*User, *Response, error) {
+func (s *UsersService) BlockUser(user int) (*Response, error) {
 	u := fmt.Sprintf("users/%d/block", user)
 
 	req, err := s.client.NewRequest("PUT", u, nil)
 	if err != nil {
-		return nil, nil, err
+		return nil, err
 	}
 
-	usr := new(User)
-	resp, err := s.client.Do(req, usr)
+	resp, err := s.client.Do(req, ioutil.Discard)
```

Comment:
Please send `nil` instead of `ioutil.Discard`, as this will save reading the body for no use (there is a `nil` check in the `s.client.Do` func for this)